lib/shared/image_viewer.dart

```diff
@@ -113,6 +113,26 @@ class _ImageViewerState extends State<ImageViewer> with TickerProviderStateMixin
     );
   }
 
+  Future<void> getImageSize(String imageUrl, context) async {
+    final ByteData data = await NetworkAssetBundle(Uri.parse(imageUrl)).load(imageUrl);
+    final Uint8List bytes = data.buffer.asUint8List();
+    final decodedImage = await decodeImageFromList(bytes);
```

Comment:
Would it help to use the `retrieveImageDimensions` method so that we don't have to download the whole image again?

Reply:
done @micahmo